tap/extensions/http/main_test.go

```diff
@@ -155,14 +154,6 @@ func TestDissect(t *testing.T) {
 
 		stop <- true
 
-		sort.Slice(items, func(i, j int) bool {
```

Comment:
where is this sort happening? or maybe it is not needed from some reason?

Reply:
@gadotroee with this change order became consistent somehow. So that sorting became redundant.